### ISSUE TITLE
Use `LIFE RIPARIAS` as project ID in metadata

### ISSUE THREAD

**Issue by peterdesmet** [riparias/crsenne-occurrences#5]
Currently `LIFE19 NAT/BE/000953`, but we opted for the shorter code to be standardized across datasets. https://www.gbif.org/dataset/49717845-86b1-4bbe-886e-4cddc1f98ebf/project

@LienReyserhove @damianooldoni I also think we have standardized metadata for the project that can be used for every dataset?

**Comment by damianooldoni** [comment]
Ineed, project ID: `LIFE RIPARIAS`
@mcoupremanne: this is the project page document we would like to use:
https://docs.google.com/document/d/1SyZmiLcpFJ2AcDaDFZfnqrndwQFaUcxr3eKHy3fJKQY/edit?usp=sharing

We wanted to use this document to create a RIPARIAS network, but GBIF preferred to not have such a network as it is a project based one. So they discouraged us (aka did not allowed us) to create it. So, this text suits perfectly for a project page within every published dataset.

**Comment by damianooldoni** [comment]
@mcoupremanne: please, wait to use the text in the Google Doc I pointed to you in previous comment as it is not ready. See https://github.com/riparias/vmm-rattenapp-occurrences/issues/11#issuecomment-1158897217

**Comment by mcoupremanne** [comment]
@damianooldoni ok I'll update it when you tell me I can use a RIPARIAS  project description

**Comment by damianooldoni** [comment]
Yes, @mcoupremanne! Still on my to do list. Sorry for the delay.

**Comment by mcoupremanne** [comment]
No worry,  took me so long to find time to go back to this publication. But it will be a bit more clean now as I can fiw a few points already. Thanks for help btw
